### PR TITLE
Added custom properties to the WP resource

### PR DESCRIPTION
**Pull request by mtakac2**

### CODE REVIEW

**Comment by ulferts** [comment]
Tests would be great

**Comment by mtakac2** [comment]
@ulferts let me add tests next week - I am having lots of unmerged PRs and some of them already have tests - If I add test now, I would have to merge it manually...

File: lib/api/v3/work_packages/work_package_representer.rb [hunk 1]
```diff
@@ -89,6 +91,11 @@ def _type
           'WorkPackage'
         end
 
+        def custom_properties
+            values = represented.work_package.custom_field_values
+            values.map { |v| { name: v.custom_field.name, format: v.custom_field.field_format, value: v.value }}
```
Comment (myabc) [inline lib/api/v3/work_packages/work_package_representer.rb:96]: @mtakac indent should be 2 spaces

Comment (mtakac2) [inline lib/api/v3/work_packages/work_package_representer.rb:96]: @myabc sorry, my bad. I don't know why sublime changed it...

**Pull request merged**